test/extended/operators/olm.go

```diff
@@ -88,7 +88,13 @@ var _ = g.Describe("[sig-operator] OLM should", func() {
 				e2e.Failf("Unable to get %s, error:%v", msg, err)
 			}
 			o.Expect(err).NotTo(o.HaveOccurred())
-			o.Expect(msg).To(o.Equal("IfNotPresent"))
+
+			// ensure that all containers in the current deployment contain the IfNotPresent
+			// image pull policy
+			policies := strings.Split(msg, " ")
```

Comment:
This test is blocking the current implementation that moves the creation and management of the PackageServer CSV downstream PR as it adds another sidecar container to the OLM Operator deployment.

Here is a sample output from that failed test:

```
: [sig-operator] OLM should have imagePullPolicy:IfNotPresent on thier deployments [Suite:openshift/conformance/parallel] expand_less
--
Run #0: Failed expand_less1sfail [github.com/openshift/origin/test/extended/operators/olm.go:91]: Expected     <string>: IfNotPresent IfNotPresent to equal     <string>: IfNotPresent | Run #0: Failed expand_less1sfail [github.com/openshift/origin/test/extended/operators/olm.go:91]: Expected     <string>: IfNotPresent IfNotPresent to equal     <string>: IfNotPresent | Run #0: Failed expand_less | 1s | fail [github.com/openshift/origin/test/extended/operators/olm.go:91]: Expected     <string>: IfNotPresent IfNotPresent to equal     <string>: IfNotPresent
Run #0: Failed expand_less1sfail [github.com/openshift/origin/test/extended/operators/olm.go:91]: Expected     <string>: IfNotPresent IfNotPresent to equal     <string>: IfNotPresent | Run #0: Failed expand_less | 1s | fail [github.com/openshift/origin/test/extended/operators/olm.go:91]: Expected     <string>: IfNotPresent IfNotPresent to equal     <string>: IfNotPresent
Run #0: Failed expand_less | 1s
fail [github.com/openshift/origin/test/extended/operators/olm.go:91]: Expected     <string>: IfNotPresent IfNotPresent to equal     <string>: IfNotPresent
```

The example output is a single string containing the "IfNotPresent IfNotPresent", which can be split up using the `" "` as the deliminator.